.github/CODEOWNERS

```diff
@@ -1 +1,27 @@
-*   @reta @anasalkouz @andrross @Bukhtawar @CEHENKLE @dblock @gbbafna @setiah @kartg @kotwanikunal @mch2 @nknize @owaiskazi19 @peternied @Rishikesh1159 @ryanbogan @saratvemulapalli @shwetathareja @dreamer-89 @tlfeng @VachaShah @dbwiddis @sachinpkale @sohami @msfroh
+# CODEOWNERS manages notifications, not PR approvals
```

Comment:
The code owners wasn't synchronized after maintainers were added and then the list was sorted by alphabetical

Reply:
Thanks for backporting the right code owners!